src/Sequence.cpp

```diff
@@ -44,6 +47,13 @@ Sequence::begin()
     KP_LOG_INFO("Kompute Sequence command now started recording");
     this->mCommandBuffer->begin(vk::CommandBufferBeginInfo());
     this->mRecording = true;
+
+    //latch the first timestamp before any commands are submitted
+    if(this->timestampQueryPool)
+        this->mCommandBuffer->writeTimestamp(
+            vk::PipelineStageFlagBits::eAllCommands,
```

Comment:
Does it need to be all or just ecompute?

Reply:
I'm not sure. I believe `eCompute` would not cover data transfer.

Reply:
Ok I can't see a disadvantage for now, should be ok to leave it for all